src/reducers/entities/channels.ts

```diff
@@ -787,6 +787,28 @@ export function channelMemberCountsByGroup(state: any = {}, action: GenericActio
     }
 }
 
+export function channelsInPolicy(state: IDMappedObjects<Channel> = {}, action: GenericAction) {
+    switch (action.type) {
+    case AdminTypes.RECEIVED_DATA_RETENTION_CUSTOM_POLICY_CHANNELS_SEARCH:
+    case AdminTypes.RECEIVED_DATA_RETENTION_CUSTOM_POLICY_CHANNELS: {
+        const nextState: IDMappedObjects<Channel> = {...state};
```

Comment:
This doesn't need to be changed now since we want to get this in, but generally Redux prefers deduplication of data, so you'd usually save the channel itself in `state.entities.channels.channels` with the others and then just store a relevant list of channel IDs here.